### ISSUE TITLE
Character count rather than word count

### ISSUE THREAD

**Issue by alantucker** [tinymce/tinymce#5746]
**Product Version**
TinyMCE 5

**I would like to know...**
I am using a limited toolbar ("undo redo bold") and was wondering if it is possible to set a character count as default at the bottom right of the editor rather than the word count? I don't want to have to rely on the user to check the character count.

**Comment by alantucker** [comment]
Hi @tylerkelly13 

Thanks for the reply and your suggestion. I definitely would like to see the option to switch between word and character count in the status bar rather than having to show the count outside of the editor.

Thanks again
Alan

**Comment by metricjs** [comment]
Hi @alantucker 

I was just thinking about this and wondered:
- Did you discover that the word count in the statusbar is a toggle, and if you click it it switches to a character count?
- What would you think of a setting that would change it from displaying word count by default to char count by default?
- What would you think of showing both in the statusbar?

Regards,
Millie

**Comment by alantucker** [comment]
Hi @metricjs 

Thanks for the reply.

I did see that there was a toggle button to switch between word and character count but this would rely on the user interacting with it (ie switching to character).

I ran into a problem where I wanted to use the editor on a field that had a max of 255 characters, in this instance a word count would have been no use to the user.

I would really like to see the option to set word or character as the default to show in the bottom right of the editor.

Thanks again, Alan.
